tests/integration/cli/test_resume.py

```
@@ -0,0 +1,258 @@
+        cast_id = "c4"
+        _journalled(project, tmp_path / "runs", cast_id)
+        (tmp_path / "runs" / cast_id / "events.jsonl").unlink()
+
+        assert main(["--resume", cast_id]) == 0
+
+        assert (project / "ran.log").read_text() == "ran-1\nran-0\n"
+
+
+class TestFramesOnDisk:
+    @staticmethod
+    def test_the_reader_skips_a_blank_line(project: Path, tmp_path: Path):
```

Ruff (0.16.1)

[warning] Missing return type annotation for staticmethod `test_the_reader_skips_a_blank_line`

Add return type annotation: `None`

(ANN205)